src/app/dashboard/experience-form/experience-form.component.html

```diff
@@ -0,0 +1,73 @@
+<form nz-form [formGroup]="experienceForm" >
+  <ng-container formArrayName="experiences">
+    <div *ngFor="let experience of experiences.controls; let idx = index" [formGroupName]="idx">
+      <div style="border: 1px solid lightslategray; border-radius: 1rem; padding: 0.5rem; margin: 0.5rem;">
+      <nz-divider [nzText]="(idx+1).toString()+') ' + (experience.value.name ?? '')"></nz-divider> 
+      <nz-form-item>
+      <nz-form-label [nzSm]="6" [nzXs]="24" nzRequired nzFor="name">Name</nz-form-label>
+      <nz-form-control [nzSm]="14" [nzXs]="24" nzErrorTip="Name is required.">
+        <input nz-input formControlName="name" id="name" />
+      </nz-form-control>
+    </nz-form-item>
+    <nz-form-item>
+      <nz-form-label [nzSm]="6" [nzXs]="24" nzFor="description" nzRequired>Description</nz-form-label>
+      <nz-form-control [nzSm]="14" [nzXs]="24" nzErrorTip="Please give description!">
+        <input
+          nz-input
+          id="description"
+          formControlName="description"
+        />
+      </nz-form-control>
+    </nz-form-item>
+    <nz-form-item>
+        <nz-form-label [nzSm]="6" [nzXs]="24" nzFor="startedFrom" nzRequired>Started from:</nz-form-label>
```

Comment:
Generally i would make this form as compact as possible.  Lets make `Staretd from` and `Gainedt at`  it in one column.